### ISSUE TITLE
How to run on a minimal installation (requirements)?

### ISSUE THREAD

**Issue by sm8ps** [giuspen/x-tile#5]
Thanks for this promising piece of software! I am excited to try it out on a minimalist installation (dCore, a Tiny Core off-spring) with LXDE/Openbox.

Though, I get the following error.

> WM = Open
> Traceback (most recent call last):
>  File "/usr/bin/x-tile", line 65, in <module>
>    x = core.XTile(core.InfoModel())
>  File "/usr/share/x-tile/modules/core.py", line 265, in **init**
>    pixbuf = gtk.gdk.pixbuf_new_from_file(filepath)
> glib.GError: Couldn't recognize the image file format for file '/usr/share/x-tile/glade/tile-vertically.svg'

So far, I have installed libgdk-pixbuf2.0-0, librsvg2-2, python-gconf, python-gtk2, python-rsvg. Do you see anything obvious that is missing?

**Comment by giuspen** [comment]
Try gtk2-engine-svg or search for gtk2 and svg in your package manager

**Comment by sm8ps** [comment]
Thanks for your quick answer! I could only try it out today but found it not to work. However, I found the solution after much research. Maybe it can be helpful for future reference. The following works with packages from the Ubuntu repositories:
1. I had to additionally install the packages python-gconf and python-rsvg with their dependencies.
2. Before launching ''x-tile'', one has to issue ''sudo /usr/lib/i386-linux-gnu/gdk-pixbuf-2.0/gdk-pixbuf-query-loaders --update-cache''.
After that, X-Tile runs beautifully on LXDE/Openbox. I am eager to fully get to try it out. Thanks a lot for yet another great piece of software after CherryTree!
